### ISSUE TITLE
[Suggestion] Option to not automatically start recording upon application launch

### ISSUE THREAD

**Issue by mhvuze** [ProxymanApp/Proxyman#996]
### Proxyman version? (Ex. Proxyman 1.4.3)
Version 2.32.1 (23210)

### macOS Version? (Ex. mac 10.14)
11.3 (20E232)

### Steps to reproduce 
When Proxyman is launched, it will automatically start recording.

### Expected behavior
I would like to suggest an option to launch Proxyman without starting the recording process as it would be really handy for certain scenarios. Perhaps Preferences → Advanced → Miscellaneous would be the ideal spot for this.

Sorry if there is already a way to do this, I wasn't able to figure it out! Thanks.

### Screenshots (optional)
–

**Comment by NghiaTranUIT** [comment]
Hey @mhvuze you can opt-out it in Preference -> General Tab -> Turn off Auto Overridden HTTP Proxy at launch 👍 

<img width="702" alt="Screen_Shot_2021-09-16_at_16_41_19" src="https://user-images.githubusercontent.com/5878421/133589660-facae5cf-17d3-4e53-ad5a-20ec27199c4d.png">

I guess I would reword it to clearer 👍

**Comment by mhvuze** [comment]
Hi @NghiaTranUIT, thanks for your quick reply!

I'm aware of the possibility to disable the proxy override at launch. However, this way traffic of other devices that are configured to use the proxy is still being recorded (since recording is automatically started when Proxyman opens, see default recording settings below).

Sorry for the confusion, I hope it's more clear now.

![](https://snipboard.io/LusNdy.jpg)

**Comment by NghiaTranUIT** [comment]
Ah I understand now 😄 I will add a checkbox below the Auto Override HTTP Proxy 👍
